ui/raidboss/data/05-shb/trial/wol-ex.js

```diff
@@ -0,0 +1,443 @@
+      condition: (data) => data.ultimateSeen,
+      infoText: {
+        en: 'Black Mage + White Mage',
+      },
+    },
+    {
+      id: 'WOLEx Summoner / Warrior',
+      netRegex: NetRegexes.startsUsing({ source: 'Spectral Summoner', id: '4F3F', capture: false }),
+      condition: (data) => data.ultimateSeen,
+      infoText: {
+        en: 'Summoner + Warrior',
+      },
+    },
+    {
+      id: 'WOLEx Spectral Bard / Dark Knight',
+      netRegex: NetRegexes.startsUsing({ source: 'Spectral Dark Knight', id: '4F3A', capture: false }),
+      condition: (data) => data.ultimateSeen,
+      infoText: {
+        en: 'Dark Knight + Bard',
+      },
+    },
+    {
+      id: 'WOLEx Spectral Ninja',
+      netRegex: NetRegexes.startsUsing({ source: 'Spectral Ninja', id: '4EFD', capture: false }),
+      infoText: {
+        en: 'Ninja',
+      },
+      run: function(data) {
+        data.ninja = true;
+      },
+    },
+    {
+      // Katon: San and Absolute Holy share markers
+      id: 'WOLEx Spectral Ninja Cleanup',
+      netRegex: NetRegexes.startsUsing({ source: 'Spectral Ninja', id: '4EFD', capture: false }),
+      delaySeconds: 30,
+      run: function(data) {
+        delete data.ninja;
+      },
+    },
+    {
+      id: 'WOLEx Suiton: San',
+      netRegex: NetRegexes.startsUsing({ source: 'Spectral Ninja', id: '4EFD', capture: false }),
+      delaySeconds: 7,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'WOLEx Katon: San',
+      netRegex: NetRegexes.headMarker({ id: '00A1', capture: false }),
+      condition: (data) => data.ultimateSeen && data.ninja,
+      delaySeconds: 0.5,
+      suppressSeconds: 2,
+      response: Responses.stack(),
+    },
+    {
+      id: 'WOLEx Perfect Decimation',
+      netRegex: NetRegexes.headMarker({ id: '0017' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Warrior Cleave on YOU',
+      },
+    },
+    {
+      id: 'WOLEx Brimstone Earth',
+      netRegex: NetRegexes.headMarker({ id: '0067' }),
+      condition: Conditions.targetIsYou(),
+      alertText: {
+        en: 'puddle on you',
+        de: 'Fläche auf DIR',
+        fr: 'Zone au sol sur VOUS',
+        ja: '自分に水溜り',
+        cn: '水球点名',
+        ko: '징 대상자',
+      },
+    },
+    {
+      id: 'WOLEx Deluge of Death Marker',
+      netRegex: NetRegexes.headMarker({ id: '0057' }),
+      condition: (data, matches) => data.ultimateSeen && data.me === matches.target,
+      alarmText: {
+        en: 'Flare on YOU',
+        de: 'Flare auf DIR',
+        fr: 'Brasier sur VOUS',
+        ja: '自分にフレア',
+        cn: '核爆点名',
+        ko: '플레어 대상자',
+      },
+      run: function(data) {
+        data.deluge = true;
+      },
+    },
+    {
+      id: 'WOLEx Absolute Holy',
+      netRegex: NetRegexes.headMarker({ id: '00A1' }),
+      condition: (data) => !data.deluge,
+      response: Responses.stackOn(),
```

Comment:
This is activating during San on the Ninja block, causing multiple callouts. This one needs to be conditioned on "Ninja not active" or something similar. Or San shouldn't have an individual trigger and this should be just a basic "stack", plus a suppress property.

Reply:
Should be fixed if you're testing this in real time. Thanks!